.gitignore

```diff
@@ -14,3 +14,15 @@
 # Dependency directories (remove the comment below to include it)
 # vendor/
 dist/
+
+# OS related files
+.DS_Store
+
+# Editor/IDE files
+.idea
```

Comment:
Don't think this goes here. You can ignore this files by the use of a global .gitignore file.

Reply:
TIL!